outlierdetect/outlierdetect.py

```diff
@@ -99,20 +99,23 @@ class MultinomialModel:
 
         def compute_outlier_scores(self, frequencies):
             """Computes the SVA outlier scores fo the given frequencies dictionary.
-        
+
             Args:
                 frequencies: dictionary of dictionaries, mapping (aggregation unit) -> (value) ->
                     (number of times aggregation unit reported value).
-            
+
             Returns:
-                dictionary mapping (aggregation unit) -> (MMA outlier score for aggregation unit).
+                - dictionary mapping (aggregation unit) -> (MMA outlier score for aggregation unit).
+                - dictionary of value counts (aggregation unit) -> (answer_choice: count, ...).
```

Comment:
```suggestion
                - dictionary mapping (aggregation unit) ->  return from _sum_frequencies: (a dictionary mapping (value) -> (number of times all aggregation units apart from agg_unit reported this value))
```

Good instinct to make it more like Ben's comment. I tweaked a bit to make it more clear.